Author: Thiago Franco de Moraes
Date: Mon, 10 Dec 2012 14:22:46 -0200
Description: Upstream has ca_smoothing integrated and Debian have it separated
in another package (python-casmoothing), so the import must be changed.
--- a/invesalius/data/surface.py
+++ b/invesalius/data/surface.py
@@ -37,7 +37,7 @@ import surface_process
 import utils as utl
 import vtk_utils as vu
 try:
-    import data.ca_smoothing as ca_smoothing
+    import ca_smoothing
 except:
     pass
 
